autocertkit/models.py

```diff
@@ -274,6 +291,22 @@ def is_finished(self):
                 return False
         return True
 
+    def group_test_method_by_status(self):
+        """Group test method by status"""
+        done, waiting, running = ([], [], [])
+        for test_method in self.test_methods:
```

Comment:
is there a more "pythonic" way to write this for-if?

Reply:
This is right python style of 'for loop'